src/pepr/operator/network/builder.ts

```diff
@@ -0,0 +1,131 @@
+import { V1APIGroup, V1NetworkPolicyPeer, V1NetworkPolicyPort } from "@kubernetes/client-node";
+import { K8s, Log, kind } from "pepr";
+
+import { Allow, UDSPackage } from "../crd";
+import { RemoteGenerated } from "../crd/generated/package-v1alpha1";
+
+let apiServerPeers: V1NetworkPolicyPeer[];
+
+export async function builder(
```

Comment:
Umm... Awkshually "Builder" is the name of a specific design pattern in OOP and may confuse people who see this function named that.